controller/wingsail/controllers.py

```diff
@@ -0,0 +1,75 @@
+from controller.common.constants import CHORD_WIDTH_MAIN_SAIL, KINEMATIC_VISCOSITY
```

Comment:
These are fine as constants, but the values used inside the class should be parameterized in the constructor and stored as member variables.